surge/api_resource.py

```diff
@@ -40,7 +40,7 @@ def _base_request(cls, method, api_endpoint, params=None, files=None, api_key=No
             elif method == "post":
                 if files is not None:
                     response = requests.post(
-                        url, auth=(api_key_to_use, ""), files=files, data=params
+                        url, auth=(api_key_to_use, ""), files=files, json=params
```

Comment:
json is almost certainly what we want -- it correctly handles nested jsons and sets the content type headers.